### ISSUE TITLE
Updates for jq 1.6 compatibility

### ISSUE THREAD

**Issue by alerque** [kislyuk/yq#46]
Given input files such as `file.txt`:
```txt
string from file
```
`f.json`:
```json
{ "foo": "bar" }
```
and `f.yaml`:
```yaml
foo: bar
```
The following command works fine in `jq`:
```sh
❯ jq -M -e -r --rawfile varname file.txt '[$varname, .foo]' -- f.json
[
  "string from file\n",
  "bar"
]
```
But the following fails in `yq`:
```sh
❯ yq -M -e -r --rawfile varname file.txt '[$varname, .foo]' -- f.yaml
usage: yq [-h] [--yaml-output] [--width WIDTH] [--version]
          jq_filter [files [files ...]]
yq: error: argument files: can't open '[$varname, .foo]': [Errno 2] No such file or directory: '[$varname, .foo]'
```

**Comment by kislyuk** [comment]
This is a jq 1.6 compatibility issue. This version was released about two months ago and I haven't had the chance to update yq to be compatible with all its options yet.

**Comment by kislyuk** [comment]
Released in v2.7.2
